stratum/hal/lib/barefoot/bf_chassis_manager_test.cc

```diff
@@ -17,7 +17,11 @@
 #include "stratum/lib/constants.h"
 #include "stratum/lib/test_utils/matchers.h"
 
-using ::stratum::test_utils::EqualsProto;
+namespace stratum {
+namespace hal {
+namespace barefoot {
```

Comment:
Don't `using` include other namespaces into the global namespace: https://google.github.io/styleguide/cppguide.html#Namespaces